Carsus_atomic_data/Custom_atomic_data.ipynb

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
<p>The shortlist flag is relevant to the format of the linelist. VALD can output lists in either "long" form or "short" form, so you need to tell STARDIS which one it is. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/stardis/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>